lib/fluent/plugin/out_dynatrace.rb

```diff
@@ -78,20 +78,26 @@ def shutdown
       #############################################
 
       def process(_tag, es)
+        log.debug('#process')
```

Comment:
`log.trace` (optionally embedded in `log.on_trace {...}`) would probably fit even better I think but both is fine.

Reply:
Great idea